components/settings/NetPyNEToolBar.js

```diff
@@ -52,48 +49,20 @@ export default class NetPyNEToolBar extends React.Component {
                         changeTab={this.props.changeTab}
                         />
                     break;
-                case 'ImportHLS':
+                case 'Import':
```

Comment:
I would still use 'ImportHLS' even if this is the only Import so we don't have to change it back in the future